### PR TITLE
Made navigation bar frontend

### PR DESCRIPTION
**Pull request by RaghuGulati**

### CODE REVIEW

**Comment by RaghuGulati** [comment]
Front end navbar

File: course/templates/course/base.html [hunk 1]
```diff
@@ -3,23 +3,253 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <title>Home Page</title>
+
+    <!-- Bootstrap CSS CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+    <!-- Our Custom CSS -->
+    <link rel="stylesheet" href="style.css">
```
Comment (SurajDadral) [inline course/templates/course/base.html:19]: Unable to find style.css file.

File: course/templates/course/base.html [hunk 2]
```diff
@@ -3,23 +3,253 @@
...
-  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-  <title>Outcome Based Course</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <title>Home Page</title>
+
+    <!-- Bootstrap CSS CDN -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
```
Comment (SurajDadral) [inline course/templates/course/base.html:17]: {% bootstrap_css %} line will do same action. So, need not to tell explicitly.

File: course/templates/course/base.html [hunk 3]
```diff
@@ -3,23 +3,253 @@
...
+    <!-- Our Custom CSS -->
+    <link rel="stylesheet" href="style.css">
+
+    <!-- Font Awesome JS -->
+    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/solid.js" integrity="sha384-tzzSw1/Vo+0N5UhStP3bvwWPq+uvzCMfrN1fEFe+xBmv1C/AtVX5K0uZtmcHitFZ" crossorigin="anonymous"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/fontawesome.js" integrity="sha384-6OIrr52G08NpOFSZdxxz1xdNSndlD4vdcf/q2myIUVO0VsqaGHJsB0RaBE01VTOY" crossorigin="anonymous"></script>
```
Comment (SurajDadral) [inline course/templates/course/base.html:23]: Try using library files after downloading them in static folder.

File: course/templates/course/base.html [hunk 4]
```diff
@@ -3,23 +3,253 @@
...
+
+ <div class="overlay"></div>
 </div>
+
+    <!-- jQuery CDN - Slim version (=without AJAX) -->
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <!-- Popper.JS -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.0/umd/popper.min.js" integrity="sha384-cs/chFZiN24E4KMATLdqdvsezGxaGsi4hLGOzlXwp5UZB1LY//20VyM2taTB4QvJ" crossorigin="anonymous"></script>
+    <!-- Bootstrap JS -->
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/js/bootstrap.min.js" integrity="sha384-uefMccjFJAIv6A+rW+L4AHf99KvxDjWSu1z9VI8SKNVmz4sk7buKt/6v9KI65qnm" crossorigin="anonymous"></script>
```
Comment (SurajDadral) [inline course/templates/course/base.html:239]: Try loading files after downloading locally in static folder.

File: course/urls.py [hunk 5]
```diff
@@ -2,7 +2,7 @@
 from .views import *
 
 urlpatterns = [
-    path("", InstituteView.as_view(), name="home"),
+    path("", home_page),
```
Comment (SurajDadral) [inline course/urls.py:5]: Add name="home"

**Pull request merged**